README.md

```diff
@@ -89,3 +89,17 @@ Below is an example of regexp patterns used for autolinking, modified in the `co
     }
 },
 ```
+
+## Examples
+
+1. Autolinking `Ticket ####:text with alphanumberic characters and spaces` to a ticket link. Use:
+  - Pattern: `(?i)(ticket )(?P<ticket_id>)( : )(?P<ticket_info>.*)`
```

Comment:
To autolink that, you need to change it to `(?i)(ticket )(?P<ticket_id>.+)(:)(?P<ticket_info>.*)` or maybe if you are sure that the ticket_id is a number you can use `(?i)(ticket )(?P<ticket_id>\d+)(:)(?P<ticket_info>.*)`